CHANGELOG.md

```diff
@@ -1,5 +1,70 @@
+- [Java] Add `getModelVersion` API to the java client (#6955, @wgottschalk)
+
+Breaking Changes:
+
+The following list of breaking changes are arranged by their order of significance within each category.
+
+- [Core] Support for Python 3.7 has been dropped. MLflow now requires Python >=3.8
+- [Recipes] MLflow Pipelines is now MLflow Recipes - a framework that enables data scientists to quickly develop high-quality models and deploy them to production
+- [Recipes] `mlflow.pipelines` APIs have been replaced with `mlflow.recipes`
+- [Tracking] Remove deprecated `list` APIs for experiments, models, and runs from Python, Java, R, and REST APIs (#6785, #6786, #6787, #6788, #6800, #6868, @dbczumar)
+- [Tracking] Remove deprecated `runs` response field from `Get Experiment` REST API response (#6541, #6524 @dbczumar)
+- [Tracking] Remove deprecated `MlflowClient.download_artifacts` API (#6537, @WeichenXu123)
+- [Tracking] Changed the behavior of environment variable handling for `MLFLOW_EXPERIMENT_NAME` such that the value is always used when creating an experiment. (#6674, @BenWilson2)
+- [Tracking] Modify tracking server configuration to use `--serve-artifacts` by default (#6502, @harupy)
+- [Tracking] Update Experiment ID generation for the Filestore backend to enable threadsafe concurrency (#7070, @BenWilson2)
+- [Tracking] Remove `dataset_name` and `on_data_{name | hash}` suffixes from `mlflow.evaluate()` metric keys (#7042, @harupy)
+- [Models / Projects] Change default to `virtualenv` instead of `conda` for MLFlow Models and MLflow Projects environment creation (#6459, #6489 @harupy)
+- [Models] Remove support for the deprecated `mlflow.keras` and `tf.estimator` flavors (#6530, @WeichenXu123)
+- [Models] Remove deprecated `mlflow.sklearn.eval_and_log_metrics` API in favor of `mlflow.evaluate()` API (#6520, @dbczumar)
+- [Models] Require `mlflow.evaluate()` model inputs to be specified as URIs (#6670, @harupy)
+- [Models] Remove the ability to specify custom metrics and artifacts within a single function call within `mlflow.evaluate()` by separating these functionalities to separate calls (#7142, @harupy)
+- [Models] Extend `PyFuncModel` spec to support `conda` and `virtualenv` subfields (#6684, @harupy)
+- [Scoring] Removed support for defining input formats using the `Content-Type` header (#6575, @tomasatdatabricks; #7254, @adriangonz)
+- [Scoring] Replace the `--no-conda` CLI option argument for native serving with `--env-manager='local'` (#6501, @harupy)
+- [Scoring] Remove public APIs for `mlflow.sagemaker.deploy()` and `mlflow.sagemaker.delete()` in favor of `MLflow deployments` CLI command `mlflow deployments -t sagemaker` (#6650, @dbczumar)
+- [Scoring] Rename input argument `df` to `inputs` in `mlflow.deployments.predict()` method (#6681, @BenWilson2)
+- [Registry / Tracking] Remove `/preview` routes for Tracking and Model Registry REST APIs (#6667, @harupy)
```

Comment:
Can we move this above the `[Tracking]` section since it's an important change that affects multiple components?

```suggestion
- [Tracking / Registry] Remove `/preview` routes for Tracking and Model Registry REST APIs (#6667, @harupy)
```